app/assets/javascripts/models/beta/story.js

```diff
@@ -427,3 +427,17 @@ export const editingStoryPropTypesShape = PropTypes.shape({
   ...storyPropTypes,
   _editing: StoryPropTypesShape
 });
+
+export const getByLabel = (stories, label) => {
+  const filteredStories = stories.filter(story => {
+    const storyLabels = Label.getNames(story.labels);
+
+    return Label.hasLabel(storyLabels, label);
+  });
+
+  return filteredStories;
+}
```

Comment:
Or you could return without creating `filteredStories`

```js
export const getByLabel = (stories, label) => 
  stories.filter(story => {
    const storyLabels = Label.getNames(story.labels);

    return Label.hasLabel(storyLabels, label);
  });
```

Reply:
Sure! But I needed change this function, now we have a request than get all stories with label. Thank you.